docs/data-access/icgc-25k-data.md

```diff
@@ -0,0 +1,100 @@
+
+The SFTP server is located at:
+
+- **Host**: `legacy-icgc-sftp.platform.icgc-argo.org`
+- **Port**: `2222`
+
+Authentication to the server is done using username and password:
+
+- **Username**: The email address that was approved for DACO access. This is the account you would use to log into the [ARGO platform](https://platform.icgc-argo.com).
+- **Password**: ICGC API Key. This is available on your ARGO Platform [profile page](https://platform.icgc-argo.com/user).
+
+You can connect to this server using any SFTP client of your choice. For a free client you consider [FileZilla](https://filezilla-project.org/download.php?type=client).
+
+## Partner Repositories with ICGC 25K File Data
+
+ICGC file data is hosted across the following repositories.
+
+If you know the specific files from ICGC that you want to access, consult the provided [Mapping File](#mapping-icgc-legacy-data-to-external-repositories) to discover which repositories host that file.
+
+### EGA
+
+Raw sequences submitted to ICGC or reprocessed apart of PCAWG are hosted across multiple EGA datasets managed by the data access committee [EGAC00001000010](https://ega-archive.org/dacs/EGAC00001000010).
+
+Upon DACO approval, the institutional email (the same email used in your DACO application, not the affiliated Gmail account) can be used to log in/sign up at [EGA-Archive](https://ega-archive.org).
+
+If the email has never been used to access EGA, please follow the password reset procedure.
+
+Access to EGA may take up to 48 hours post DACO approval.
+
+### PDC
+
+Due to data regulation policies, TCGA affiliated data from the PCAWG study are saved in PDC and controlled under dbGap study [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8).
+
+DACO approval does not include dbGap study [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8) and will not grant access to said files.
+
+Access can be requested by following instructions in [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8).
+
+To access data, navigate to [ICGC Bionimbus](https://icgc.bionimbus.org/files) and follow the login prompt using eRA commons ID and password.
+
+### GDC
+
+Due to data regulation policies, TCGA raw sequencing data submitted to ICGC or used in PCAWG are saved in GDC and controlled under dbGap study [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8). DACO approval does not include dbGap study [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8) and will not grant access to said files.
+
+Access can be requested by following instructions in [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8).
+To access data, navigate to [GDC Portal](https://portal.gdc.cancer.gov) and follow the login prompt using eRA commons ID and password.
```

Comment:
This is the same link than for PDC - do we need to keep both sections then? 

Do we care PDC vs GDC? Or should we say "TCGA affiliated data from PCAWG are available under dbGap study [phs000178](https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000178.v11.p8)."

Reply:
wonderful, thanks!